src/evernodetwo.c

```diff
@@ -76,6 +78,10 @@ int64_t hook(uint32_t reserved)
             if (state(SBUF(host_addr), SBUF(STP_HOST_ADDR)) == DOESNT_EXIST)
                 rollback(SBUF("Evernode: This host is not registered."), 1);
 
+            TOKEN_ID_KEY(host_addr);
```

Comment:
Is this should be likeTOKEN_ID_KEY((uint8_t *)(host_addr + HOST_TOKEN_ID_OFFSET)) ?